sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -12,118 +18,82 @@
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts",
-    "pack": "npm pack 2>&1",
-    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
-    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
-    "test": "npm run clean && npm run build:test && npm run unit-test",
-    "unit-test:browser": "dev-tool run test:browser",
-    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
-    "unit-test": "cross-env TEST_MODE=playback && npm run unit-test:node && npm run unit-test:browser"
-  },
-  "sideEffects": false,
-  "autoPublish": false,
   "dependencies": {
-    "@azure/core-rest-pipeline": "^1.9.2",
-    "@azure-rest/core-client": "^1.0.0",
-    "@azure/core-auth": "^1.3.0",
-    "@azure/core-lro": "^2.2.4",
-    "@azure/core-paging": "^1.2.0",
+    "@azure-rest/core-client": "^1.4.0",
+    "@azure/core-auth": "^1.6.0",
+    "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
-    "tslib": "^2.2.0"
+    "tslib": "^2.6.2",
+    "@azure/core-lro": "^3.0.0",
+    "@azure/abort-controller": "^2.1.2",
+    "@azure/core-paging": "^1.5.0"
   },
   "devDependencies": {
-    "@azure/core-util": "^1.0.0",
-    "@azure/dev-tool": "^1.0.0",
-    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^4.0.1",
-    "@azure-tools/test-utils": "^1.0.1",
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@microsoft/api-extractor": "^7.31.1",
-    "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
-    "@types/node": "^18.0.0",
-    "chai": "^4.2.0",
-    "cross-env": "^7.0.2",
     "dotenv": "^16.0.0",
-    "eslint": "^8.0.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-json-preprocessor": "^0.3.3",
-    "karma-json-to-file-reporter": "^1.0.1",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-mocha": "^2.0.1",
-    "karma-source-map-support": "~1.4.0",
-    "karma-sourcemap-loader": "^0.3.8",
-    "karma": "^6.2.0",
-    "mkdirp": "^3.0.1",
-    "mocha": "^10.0.0",
-    "nyc": "^17.0.0",
+    "@microsoft/api-extractor": "^7.40.3",
+    "@types/node": "^18.0.0",
+    "eslint": "^8.55.0",
+    "prettier": "^3.2.5",
     "rimraf": "^5.0.5",
-    "source-map-support": "^0.5.9",
+    "mkdirp": "^3.0.1",
     "typescript": "~5.5.3",
-    "ts-node": "^10.0.0"
+    "autorest": "latest",
+    "source-map-support": "^0.5.9",
+    "uglify-js": "^3.4.9",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0"
```

Comment:
seems missing cjs dependencies here

Reply:
those are test dev dependencies, you need to enable generateTest: true to get them generated.